programs/uxd/tests/integration_tests/api/program_uxd/procedures/process_deploy_program.rs

```diff
@@ -0,0 +1,65 @@
+use solana_program_test::ProgramTestContext;
+use solana_sdk::signature::Keypair;
+use solana_sdk::signer::Signer;
+
+use crate::integration_tests::api::program_spl;
+use crate::integration_tests::api::program_uxd;
+
+pub async fn process_deploy_program(
+    program_test_context: &mut ProgramTestContext,
+    program_keys: &program_uxd::accounts::ProgramKeys,
+    payer: &Keypair,
+) -> Result<(), String> {
+    // Use restictive default values for all tests
+    // Can be modified in individual test cases through edits
+    // This forces all tests be explicit about their requirements
+    let redeemable_mint_decimals = 6;
+    let redeemable_global_supply_cap = 0;
+    let identity_depository_redeemable_amount_under_management_cap = 0;
+    let identity_depository_minting_disabled = true;
```

Comment:
Could be interesting to have theses as optional parameters for this function. So every test can setup the initial values easily without having to call `process_edit_controller` and `process_edit_identity_depository`.

Reply:
After adding credix/merucial, there is like 15 parameters between fees and caps and minting_disabled 😆